docs/about/participating.rst

```diff
@@ -4,15 +4,15 @@ Participating
 
 CIViC Principles
 ~~~~~~~~~~~~~~~~
-- **Community consensus.** The interpretations of clinical actionability required to enable precision medicine should be freely available and openly discussed across a diverse community. To facilitate consensus building, the interface must support direct contribution from members of the community.
+- **Community consensus.** The interpretations of clinical actionability required to enable precision medicine should be freely available and openly discussed across a diverse community. To facilitate consensus building, the interface must support direct contribution and conversation from members of the community.
```

Comment:
Honestly, I might say "discussion" rather than "conversation" but I don't want that minor detail to waylay this pull request. Added a commit that could be undone, ideally. Should have done a pull request probably.